components/config/caching.rst

```diff
@@ -1,26 +1,27 @@
 .. index::
    single: Config; Caching based on resources
 
-Caching Based on Resources
+Caching based on Resources
```

Comment:
I don't understand the twisted logic behind title casing, but are we sure about this change?